--- axis-1_4/src/org/apache/axis/i18n/ProjectResourceBundle.java.orig	2008-01-11 00:17:49.000000000 +0000
+++ axis-1_4/src/org/apache/axis/i18n/ProjectResourceBundle.java	2008-01-11 00:19:22.000000000 +0000
@@ -357,14 +357,6 @@
         return resourceName;
     }
 
-    /**
-     * Clears the internal cache
-     */
-    public static void clearCache()
-    {
-        bundleCache.clear();
-    }
-    
     public String toString() {
         return resourceName;
     }
